bot/README.md

```diff
@@ -30,7 +30,7 @@ code-review-bot --taskcluster-secret=path/to/secret
 Configuration
 -------------
 
-The code review bot is configured through the [Taskcluster secrets service](https://tools.taskcluster.net/secrets)
+The code review bot is configured through the [Taskcluster secrets service](https://community-tc.services.mozilla.com/secrets)
```

Comment:
This should be firefox-ci

Reply:
Well both... I guess the Heroku TC clients  will be on commuity-tc

Reply:
It might be better to use firefox-ci for everything deployed, and community-tc only for CI, otherwise we have to duplicate secrets

Reply:
Done